projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/dimension/scd1/02-handle-quality-checks.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import re
+
+from vdk.api.job_input import IJobInput
+
+SQL_FILES_FOLDER = (
+    os.path.dirname(os.path.abspath(__file__)) + "/02-requisite-sql-scripts"
+)
+
+
+def run(job_input: IJobInput):
+    job_arguments = job_input.get_arguments()
+
+    check = job_arguments.get("check")
```

Comment:
I think we should allow users to omit passing check (and assume it's None). 

```suggestion
    check = job_arguments.get("check", None)
```

Reply:
Do we need passing a default value as None? The default behavior of dict.get("key") is to return None if the key is not present.

Reply:
Oh, I forgot, In this case I guess no.